MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/IEnggDiffractionView.h

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 #include <vector>
+#include <QStringList>
```

Comment:
@FedeMPouzols it was needed because the Mock file wasn't recognising the `QStringList`, which was then fixed by this include it in view header